sql/src/main/java/io/crate/executor/transport/task/elasticsearch/ESGetTask.java

```diff
@@ -96,8 +97,29 @@ public ESGetTask(UUID jobId,
             extractors.add(SYMBOL_TO_FIELD_EXTRACTOR.convert(symbol, ctx));
         }
 
-        final FetchSourceContext fsc = new FetchSourceContext(ctx.referenceNames());
 
+        boolean containsSysColumn = false;
+        for (Reference ref : ctx.references()) {
+            if (ref.ident().columnIdent().isSystemColumn()) {
+                if (ref.ident().columnIdent().name().equals("_raw")
+                        || ref.ident().columnIdent().name().equals("_doc")) {
+                    containsSysColumn = true;
+                    break;
+                } else {
+                    containsSysColumn = false;
+                    break;
+                }
+            } else {
+                containsSysColumn = false;
+            }
+        }
+
+        final FetchSourceContext fsc;
+        if (containsSysColumn) {
+            fsc = new FetchSourceContext(Strings.EMPTY_ARRAY);
+        } else {
+            fsc = new FetchSourceContext(ctx.referenceNames());
```

Comment:
build referencenames with the above loop and remove method. this will allow you to just insert non sys cols into the includes array